Index: setup.py
===================================================================
--- a/setup.py
+++ b/setup.py	2014-07-27 21:04:27.325945945 +0200
@@ -244,7 +244,7 @@
             self.define_macros.append(('HAVE_CURL_SSL', 1))
         if not self.libraries:
             self.libraries.append("curl")
-        
+        self.libraries.append("crypto")
         # Add extra compile flag for MacOS X
         if sys.platform[:-1] == "darwin":
             self.extra_link_args.append("-flat_namespace")
