diff -up python2/setup.py.1~ python2/setup.py
--- python2/setup.py.1~	2020-02-24 19:21:39.462620688 +0100
+++ python2/setup.py	2020-02-24 19:23:04.374619450 +0100
@@ -344,6 +344,8 @@ manually. For other SSL backends please
         if not self.libraries:
             self.libraries.append("curl")
 
+        self.libraries.append("crypto")
+
         # Add extra compile flag for MacOS X
         if sys.platform.startswith('darwin'):
             self.extra_link_args.append("-flat_namespace")
